recipe/meta.yaml

```diff
@@ -16,7 +16,7 @@ source:
     - 8c4f77a9f35eb97a47a09a9555c27c9ca350bd5e.patch
 
 build:
-  number: 4
+  number: 5
```

Comment:
Does this propagate though to all split packages?

Reply:
Yes!